Added the deprecation information of DTM (#175)
SoftwareAG/c8y-releasenotes: content/release-10-17-0/announcements-10-17-0.md

As of Cumulocity IoT 10.16, Machine Learning Engine and Machine Learning Workbench are deprecated, and we will end further development of these components. After Cumulocity IoT 10.17, both will be removed from the platform. If you have existing (production) projects which rely on MLE and/or MLW applications, then Software AG is committed to work together with you to find an appropriate transition path. If we have not yet been in contact for this matter, feel free to contact us at [c8y_mlw_mle_sunsetting@softwareag.com](mailto:c8y_mlw_mle_sunsetting@softwareag.com).

For details, see the announcement in the release notes for [release 10.16](/release-10-16-0/announcements-10-16-0).

### Digital Twin Manager

> #### Planned

Reviewer: ```suggestion
#### Implemented
```

> ##### Deprecation of Assets move feature

Reviewer: ```suggestion
##### Deprecation of asset navigation
```

Reviewer: @spau6 In the text you say that the asset navigation is deprecated, I wanted to make it consistent :-). If we call it asset move feature here we should call it the same in the text itself.

Reviewer 2: Rephrasing this as 'Deprecation of asset navigation using Left navigation' in both header and text would be a better fit as we have replaced asset navigation using left nav with navigation using asset tree and subassets.

Reviewer: How about saying "Deprecation of former asset navigation" in the header and "Deprecation of asset navigation using left navigation" in the text?

> In the 10.17 version release, we have introduced an improved way of viewing the hierarchical representation of an asset with our new Asset Tree feature. This will replace the deprecated Asset Navigator feature which allowed users to navigate to different assets and move them to another asset hierarchy. With the new Asset Tree feature, users can easily view the hierarchical representation of an asset and navigate to different hierarchies.

Reviewer 3: ```suggestion
In the 10.17 version release, we have introduced an improved way of viewing the hierarchical representation of an asset with our new Asset tree feature. This will replace the deprecated Asset navigator feature which allowed users to navigate to different assets and move them to another asset hierarchy. With the new Asset tree feature, users can easily view the hierarchical representation of an asset and navigate to different hierarchies.
```

Reviewer: @MWindel So "Asset" should be capitalized?

> In the 10.17 version release, we have introduced an improved way of viewing the hierarchical representation of an asset with our new Asset tree feature. This will replace the deprecated Asset navigator feature which allowed users to navigate to different assets and move them to another asset hierarchy. With the new Asset tree feature, users can easily view the hierarchical representation of an asset and navigate to different hierarchies.

Reviewer: ```suggestion
The former asset navigation, which allowed users to navigate to different assets and move them to another asset hierarchy, is deprecated. With release 10.17, we introduce an improved way of viewing the hierarchical representation of an asset with our new asset tree feature which allows users to easily view the hierarchical representation of an asset and navigate to different hierarchies.
```

> Asset navigator feature is deprecated where in user can view the hierarchical representation of the asset, navigate to different asset and can move the assets to supporting asset hierarchy. As part of 10.17 DTM release Asset tree feature is introduced so user can view the hierarchical representation of an asset in a more precise way also user can navigate to differnt hierarchies of an asset. Asset movement will be supported in 10.18 release of DTM.

Reviewer 2: ```suggestion
In the 10.17 version release, we have introduced an improved way of viewing the hierarchical representation of an asset with our new Asset Tree feature. This will replace the deprecated Asset Navigator feature which allowed users to navigate to different assets and move them to another asset hierarchy. With the new Asset Tree feature, users can easily view the hierarchical representation of an asset and navigate to different hierarchies. Asset movement will be supported in the next version release, 10.18.
```

> In the 10.17 version release, we have introduced an improved way of viewing the hierarchical representation of an asset with our new Asset Tree feature. This will replace the deprecated Asset Navigator feature which allowed users to navigate to different assets and move them to another asset hierarchy. With the new Asset Tree feature, users can easily view the hierarchical representation of an asset and navigate to different hierarchies. Asset movement will be supported in the next version release, 10.18.

Author: Assets move feature is deprecated in 10.17
We will add the same feature in 10.18

> In the 10.17 version release, we have introduced an improved way of viewing the hierarchical representation of an asset with our new Asset Tree feature. This will replace the deprecated Asset Navigator feature which allowed users to navigate to different assets and move them to another asset hierarchy. With the new Asset Tree feature, users can easily view the hierarchical representation of an asset and navigate to different hierarchies. Asset movement will be supported in the next version release, 10.18.

Author: Updated the content

#### Implemented

##### Deprecation of former asset navigation

> Deprecation of asset navigation using left navigation, which allowed users to navigate to different assets and move them to another asset hierarchy. With release 10.17, we introduce an improved way of viewing the hierarchical representation of an asset with our new asset tree feature which allows users to easily view the hierarchical representation of an asset and navigate to different hierarchies.

Reviewer: ```suggestion
The former asset navigation using left navigation, which allowed users to navigate to different assets and move them to another asset hierarchy, is deprecated. With release 10.17, we introduce an improved way of viewing the hierarchical representation of an asset with our new asset tree feature which allows users to easily view the hierarchical representation of an asset and navigate to different hierarchies.
```
